packages/x-components/src/x-modules/scroll/components/__tests__/scroll.spec.ts

```diff
@@ -29,37 +31,41 @@ function renderBaseIdScroll({
     }
   );
 
-  const wrapper = wrapperContainer.findComponent(BaseIdScroll);
+  const wrapper = wrapperContainer.findComponent(Scroll);
   const scrollElement: HTMLElement = wrapperContainer.find(
     getDataTestSelector('base-scroll')
   ).element;
   jest.spyOn(scrollElement, 'clientHeight', 'get').mockImplementation(() => clientHeight);
   jest.spyOn(scrollElement, 'scrollHeight', 'get').mockImplementation(() => scrollHeight);
 
+  /* Because the scroll mixins uses a $nextTick in the mounted hook, we have to add a wait here
+  so it has really finish mounting */
+  await wrapper.vm.$nextTick();
+
   return {
     wrapper,
     scrollElement,
-    async scroll(options: { to: number; durationMs: number }) {
-      scrollElement.scrollTop = options.to;
+    async scroll({ to, durationMs }) {
+      scrollElement.scrollTop = to;
       wrapper.trigger('scroll');
-      jest.advanceTimersByTime(options.durationMs);
+      jest.advanceTimersByTime(durationMs);
       await wrapper.vm.$nextTick();
     }
   };
 }
 
-describe('testing Base Scroll Component', () => {
+describe('testing Scroll Component', () => {
   beforeAll(jest.useFakeTimers);
   afterEach(jest.clearAllTimers);
 
-  it('renders default slot contents', () => {
-    const { wrapper } = renderBaseIdScroll({
+  it('renders default slot contents', async () => {
+    const { wrapper } = await renderScroll({
       template: `
-        <BaseIdScroll :id="id">
+        <Scroll :id="id">
           <div data-test="content-scroll">
             <p>scroll content</p>
           </div>
-        </BaseIdScroll>`,
+        </Scroll>`,
       id: 'scrollResults'
```

Comment:
This `id` parameter already has a `scrollResults` value.

Reply:
Changed it. It is all about being explicit